packages/base/tests/functional/basics/Random.lua

```diff
@@ -24,6 +24,148 @@
 --          Rodrigo Reis Pereira
 -------------------------------------------------------------------------------------------
 
+local TestDistributions = Model{
+	finalTime = 1000,
+	distribution = Choice{"exponential", "normal", "lognormal", "beta"},
+	minChart = 0,
+	maxChart = 10,
+	stepChart = 0.5,
+
+	beta = function()
+		return Random{distrib = "beta"}
+	end,
+	exponential = function()
+		return Random{distrib = "exponential"}
+	end,
+	normal = function()
+		return Random{distrib = "normal", mean = 0}
+	end,
+	lognormal = function()
+		return Random{distrib = "lognormal", sd = 0.5}
+	end,
+	init = function(model)
+		Random{seed = 10}
+		model.data = {}
+		model.expected = {}
+		model.random = model[model.distribution](model)
+
+		local file = filePath("test/"..model.distribution.."_data.csv")
+		model.csv = file:read()
+
+		model.timer = Timer{
+			Event{
+				action = function()
+					table.insert(model.data, model.random:sample())
+					table.insert(model.expected, model.csv[model.timer:getTime()].x)
+				end
+			},
+			Event{
+				start = model.finalTime,
+				priority = "verylow",
+				action = function()
+					local dfData = model:density(model.data, model.minChart, model.maxChart, model.stepChart)
+					local dfExpected = model:density(model.expected, model.minChart, model.maxChart, model.stepChart)
+					local minimum
+					if #dfData.x < #dfExpected.x then
+						minimum = dfData.x
+					else
+						minimum = dfExpected.x
+					end
+					local tme = {}
+					local r = {}
+					local x = {}
+					for i = 1, #minimum do
+						table.insert(tme, dfData.density[i])
+						table.insert(r, dfExpected.density[i])
+						table.insert(x, minimum[i])
+					end
+					local df = DataFrame{TerraME = tme, R = r, x = x}
+					Chart{target = df, select = {"TerraME", "R"}, xAxis = "x", yLabel = "density", title = model.distribution}
```

Comment:
This chart is unnecessary as there is no `UnitTest:assertSnapshot` associated to it.

Reply:
Should I remove this chart or use UnitTest:assertSnapshot?

Reply:
Remove the Chart. The assertEquals are enough.